lighthouse-core/report/report-generator.js

```diff
@@ -101,6 +101,15 @@ class ReportGenerator {
     // !value
     Handlebars.registerHelper('not', value => !value);
 
+    // value == value2?
+    Handlebars.registerHelper('if_not_eq', function(conditional, options) {
+      if (conditional !== options.hash.equals) {
+        return options.fn();
```

Comment:
and no clue if it's required, but all the examples have `options.fn(this)` and `options.inverse(this);`